android/.gitignore

```diff
@@ -1,2 +1,90 @@
 .gradle
 .externalNativeBuild
+
+# https://github.com/github/gitignore/blob/master/Android.gitignore
```

Comment:
It feels a bit strange to put this in every project, instead of in a system-wide or user-specific .gitignore file.

Reply:
Makes sense .. I've pulled out the .gitignore file ..